README.md

```diff
@@ -55,7 +55,7 @@ Now we should be ready to run the server:
 Other things you can do:
 
 * Run the test suite with `npm test`
-* Check your code style with npm run jscs (according to the Google style guide)
+* Check your code style with `node_modules/eslint/bin/eslint.js census/<filename.js>` using the eslint config `/.eslintrc`.
```

Comment:
The codebase is currently in transition, so not all files will pass linting. Now might be a good time to go through and align all files with the linting standards, then I'd be more comfortable with adding an `npm run` command and posttest. I like the idea of only checking linting if all tests pass though.